test/sqlsmith/Dockerfile

```diff
@@ -28,6 +28,7 @@ ADD https://api.github.com/repos/MaterializeInc/sqlsmith/git/refs/heads/master v
 # Build SQLsmith
 RUN git clone --depth=1 --single-branch --branch=master https://github.com/MaterializeInc/sqlsmith \
     && cd sqlsmith \
+    && git checkout efba94d292c7f32c46e2637a3e05d6f95acf944b \
```

Comment:
Optional: Shall we use a tag?

Reply:
I think no real need for versions yet, but it's good to have a fixed SQLsmith version in a version of Materialize code, especially for reproducing issues.